### PR TITLE
avs-device-sdk: install Alexa sample application

### PR DESCRIPTION
**Pull request by TSAnsari**
Install Alexa sample application and its following dependencies,

- gstreamer
- portaudio

In the future we can decide to keep or discard the Sample app.
If we decide to discard the sample app then we should make sure
to remove these dependencies, unless they are used by something
else.

Signed-off-by: Tariq Ansari <tansari@luxoft.com>

### CODE REVIEW

**Comment by thenor** [comment]
Looks good to me, but I don't feel I have the domain knowledge to be able to approve.

File: recipes-alexa/avs-device-sdk/avs-device-sdk.inc [hunk 1]
```diff
@@ -27,9 +27,14 @@ inherit pkgconfig cmake
...
-                  -DACSDK_LATENCY_LOG=${AAC_LATENCY_LOGS}"
+                  -DACSDK_LATENCY_LOG=${AAC_LATENCY_LOGS} \
+                  -DGSTREAMER_MEDIA_PLAYER=ON \
+                  -DPORTAUDIO=ON \
+                  -DPORTAUDIO_LIB_PATH=${STAGING_DIR_HOST}${libdir}/libportaudio.so \
+                  -DPORTAUDIO_INCLUDE_DIR=${STAGING_DIR_HOST}${includedir} \
+                 "
 
-FILES_${PN} = "${libdir}"
+FILES_${PN} = "${libdir} ${bindir}/SampleApp"
```
Comment (sashko) [inline recipes-alexa/avs-device-sdk/avs-device-sdk.inc]: separate package, please

File: recipes-alexa/avs-device-sdk/avs-device-sdk.inc [hunk 2]
```diff
@@ -10,7 +10,7 @@ HOMEPAGE = "https://developer.amazon.com/avs/sdk"
 LICENSE = "Apache-2.0"
 LIC_FILES_CHKSUM = "file://LICENSE.txt;md5=d92e60ee98664c54f68aa515a6169708"
 
-DEPENDS = "curl nghttp2 sqlite3 openssl"
+DEPENDS = "curl nghttp2 sqlite3 openssl gstreamer1.0 gstreamer1.0-plugins-base portaudio-v19"
```
Comment (sashko) [inline recipes-alexa/avs-device-sdk/avs-device-sdk.inc]: `portaudio-v19` dependancy for the `SampeApp` only, please

File: recipes-alexa/avs-device-sdk/avs-device-sdk.inc (hunk 1 above)
Comment (martin-ejdestig) [reply recipes-alexa/avs-device-sdk/avs-device-sdk.inc]: Or a package config?

Comment (TSAnsari) [reply recipes-alexa/avs-device-sdk/avs-device-sdk.inc]: @sashko 
Do you mean something like below?

PACKAGES += "${PN}-sampleapp"
FILES_${PN}-sampleapp = "${bindir}/SampleApp"

Comment (sashko) [reply recipes-alexa/avs-device-sdk/avs-device-sdk.inc]: yes

**Pull request closed without merging**